src/constants.js

```diff
@@ -176,6 +176,7 @@ export const DEFAULT_PATH_STATIC_PREVIEW = 'platform/preview';
 export const DEFAULT_HOSTNAME_API = 'https://api.box.com';
 export const DEFAULT_HOSTNAME_STATIC = 'https://cdn01.boxcdn.net';
 export const DEFAULT_HOSTNAME_UPLOAD = 'https://upload.box.com';
```

Comment:
Can we leave a comment here why we are leaving the const for `https://upload.box.com` and not removing it?

Reply:
^duplicate of @danubevictoria comment